src/components/backend-ai-login.ts

```diff
@@ -1652,11 +1658,16 @@ export default class BackendAILogin extends BackendAIPage {
                       @keyup="${this._submitIfEnter}"
                       >
                   </mwc-textfield>
-                  <mwc-textfield type="password" id="id_password" autocomplete="current-password"
-                      label="${_t('login.Password')}" icon="vpn_key"
-                      value="${this.password}"
-                      @keyup="${this._submitIfEnter}">
-                  </mwc-textfield>
+                  <div class="horizontal flex layout">
+                    <mwc-textfield type="password" id="id_password" autocomplete="current-password"
+                        label="${_t('login.Password')}" icon="vpn_key"
+                        value="${this.password}"
+                        @keyup="${this._submitIfEnter}">
+                    </mwc-textfield>
+                    <mwc-icon-button-toggle off onIcon="visibility" offIcon="visibility_off"
+                        @click="${(e) => this._togglePasswordVisibility(e.target)}">
+                    </mwc-icon-button-toggle>
+                  </div>
```

Comment:
I think it would be more natural if the toggle button is placed inside the text field (this is not related to markup, just referring to the styling).

Are you aware of any other methods to style the button?

Reply:
I think so too... So I tried to find style at material design, but material design's textField have iconTrailing property but icon button isn't support...

I think same problem at user's info change
![스크린샷 2023-07-26 20-09-54](https://github.com/lablup/backend.ai-webui-ossca-2023/assets/28584221/de6b9177-d929-48e2-a26e-de782d757f78)

if we use ant design instead of material design, we can apply that toogle button in placed inside the text field!!
ex) [ant design input](https://ant.design/components/input)

Reply:
well, I guess it's a challenge to convert "material" into "ant". 

I recommend you to resolve this issue when changing the web component to the react component.